src/content/app/genome-browser/components/browser-sidebar-modal/services/browser-sidebar-modal-storage-service.ts

```diff
@@ -64,4 +64,4 @@ export class TrackPanelStorageService {
   }
 }
 
-export default new TrackPanelStorageService(storageService);
+export default new BrowserSidebarModalStorageService(storageService);
```

Comment:
Does this perhaps look rather like `GenomeBrowserBookmarksService`?

Reply:
✅